payroll-processor-functions/src/payroll-processor-functions/Features/Payrolls/PayrollTrigger.cs

```diff
@@ -0,0 +1,156 @@
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.Azure.WebJobs;
+using Microsoft.Azure.WebJobs.Extensions.Http;
+using Microsoft.Extensions.Logging;
+using Microsoft.WindowsAzure.Storage.Queue;
+using Microsoft.WindowsAzure.Storage.Table;
+using PayrollProcessor.Functions.Features.Employees;
+using PayrollProcessor.Functions.Infrastructure;
+using System;
+using System.Threading.Tasks;
+
+namespace PayrollProcessor.Functions.Features.Payrolls
+{
+    public class PayrollTrigger
+    {
+        [FunctionName("Payroll_GetAll")]
+        public async Task<IActionResult> GetPayrolls(
+            [HttpTrigger(AuthorizationLevel.Anonymous, "GET", Route = "payrolls")] HttpRequest req,
```

Comment:
I'm using the `Route` property of the `HttpTrigger` attribute to give the Functions endpoints more of a "REST-ish" look and feel.